src/ui/server/routes/insurance/check-your-answers/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { get, post } from '../../../test-mocks/mock-router';
+import { ROUTES } from '../../../constants';
+
+import { get as getEligibilityCheckYourAnswers, post as postEligibilityCheckYourAnswers } from '../../../controllers/insurance/check-your-answers/eligibility';
```

Comment:
same as previous comment if we could prefix with `checkYourAnswers` first so `getCheckYourAnswersEligibility` etc